spec/support/pages/audit/content_audit_tool.rb

```diff
@@ -4,4 +4,8 @@ class ContentAuditTool
   def audit_content_item
     AuditContentItemPage.new
   end
+
+  def filter_audit_list_page
```

Comment:
I think we should try and adopt terms closest to how our users would describe the pages of the tool. This page has the title 'Audit content' in the active tab? Maybe this could be `audit_content_page`?

Reply:
Addressed in https://github.com/alphagov/content-performance-manager/pull/398/commits/64467801e89fd50bc662f181cb73bbd93e6d0728